libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ReleaseHelper.java

```diff
@@ -58,6 +77,27 @@ public SubTable makeRows(Release release) throws SW360Exception {
 
     private void addFieldValueToRow(List<String> row, Release._Fields field, Release release) throws SW360Exception {
         switch (field) {
+            case COMPONENT_ID:
+                // first, add data for given field
+                row.add(fieldValueAsString(
+                        release.getFieldValue(org.eclipse.sw360.datahandler.thrift.components.Release._Fields.ID)));
+
+                // second, add joined data - but only if wanted
+                // remark that headers have already been added accordingly
+                if (extendedByAdditionalData) {
+                    try {
+                        Component component = cClient.getComponentById(release.componentId, this.user);
```

Comment:
Okay, I added some batch loading for components of releases and of linked releases. Hope that this does not slow the performance too much now.